delivery-service/src/main/java/bi/seke/deliveryservice/entities/PackageEntity.java

```diff
@@ -1,33 +1,17 @@
 package bi.seke.deliveryservice.entities;
 
-import lombok.Data;
-import org.springframework.data.cassandra.core.mapping.CassandraType;
-import org.springframework.data.cassandra.core.mapping.PrimaryKey;
-import org.springframework.data.cassandra.core.mapping.Table;
-
 import java.util.UUID;
 
-@Table
-@Data
+
 public class PackageEntity {
-    @PrimaryKey
     private PackagePK pk;
```

Comment:
please check the package for date to datetime, because we can use them for statistics